web/src/main/bvmui/src/components/Card.vue

```diff
@@ -1,28 +1,44 @@
 <template>
-    <div class="bvm-card" @click="$emit('click', $event)">
-        <div class="bvm-card-description">{{data.description}}</div>
-        <div class="bvm-card-status">
-            <h1 >{{data.state}}</h1>
-        </div>
-        <div class="bvm-card-content">
-            <ul>
-                <li v-if="data.totalDiskSpace > 0">
-                    <span>Free space: {{$library.formatBytes(data.freeDiskSpace)}} - {{$library.formatPercent(data.freeDiskSpace, data.totalDiskSpace)}} %</span>
-                </li>
-                <li v-if="data.totalDiskSpace > 0">
-                    <span>Total space: {{$library.formatBytes(data.totalDiskSpace)}}</span>
-                </li>
-                <li>
-                    <span>Sampled at {{new Date(data.lastScan)}}.</span>
-                </li>
-            </ul>
-        </div>
-    </div>
+    <v-card class="mx-auto" :color="statuscolor" raised dark @click="$emit('click', $event)">
+        <v-list-item two-line>
+        <v-list-item-content>
+            <v-list-item-title class="headline">{{data.description}}</v-list-item-title>
+            <v-list-item-subtitle>{{$library.formatDate(data.lastScan)}}</v-list-item-subtitle>
+        </v-list-item-content>
+        </v-list-item>
+        <v-card-text>
+            <p class="text-center display-3">{{$library.formatPercent(data.totalDiskSpace - data.freeDiskSpace, data.totalDiskSpace)}}&#37;</p>            
```

Comment:
unfortunately totalDiskSpace and totalDiskSpace are not always available.
I won't put this information in the foreground.
The most useful info are:
- the ID of the bookie (hostname:port)
- The status: available, readonly, down

Reply:
ok